JS/main.js

```diff
@@ -1,31 +1,30 @@
-"use strict";
-let books = [];
-const listSection = document.querySelector(".list-section");
-const bookTitle = document.querySelector("#title");
-const bookAuthor = document.querySelector("#author");
-const theForm = document.querySelector("form");
+const books = [];
+const listSection = document.querySelector('.list-section');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const theForm = document.querySelector('form');
 
 // adding items in the books array
 
 function addBookItem(bookItem) {
-  const bookBlock = document.createElement("div");
-  bookBlock.classList.add("bookBlock");
+  const bookBlock = document.createElement('div');
+  bookBlock.classList.add('bookBlock');
 
-  const removeBtn = document.createElement("button");
-  removeBtn.classList.add("remove-btn");
-  removeBtn.innerText = "remove";
+  const removeBtn = document.createElement('button');
+  removeBtn.classList.add('remove-btn');
+  removeBtn.innerText = 'remove';
 
-  const underLine = document.createElement("hr");
+  const underLine = document.createElement('hr');
 
-  bookBlock.innerHTML = `<p class="book-title">${bookItem.title}</p>
-			<p class="book-Author">${bookItem.author}</p>`;
+  bookBlock.innerHTML = `<p class='book-title'>${bookItem.title}</p>
+<p class='book-Author'>${bookItem.author}</p>`;
   bookBlock.appendChild(removeBtn);
   bookBlock.appendChild(underLine);
   listSection.appendChild(bookBlock);
-  let x = Array.from(document.querySelectorAll(".bookBlock"));
+  const x = Array.from(document.querySelectorAll('.bookBlock'));
 
-  document.querySelectorAll(".remove-btn").forEach((btn, i) => {
-    btn.addEventListener("click", () => {
+  document.querySelectorAll('.remove-btn').forEach((btn, i) => {
+    btn.addEventListener('click', () => {
       x[i].remove();
```

Comment:
- [ ] You have successfully removed a bookBlock(div) together with the underline from the page. From the requirements of this project, we are required to also remove a book from the collection(Books array) ie when click on the remove button, a book is supposed to disappear from the page and the right book is also removed from the collection. Please let's fix this.